Gemfile

```diff
@@ -2,3 +2,4 @@ source 'https://rubygems.org'
 
 gem 'rake'
 gem 'gemika'
+gem 'rspec', '>=2.13.0'
```

Comment:
This Gemfile should not have a dependency to rspec. It is only used to freeze the versions of gemika and rake (gemika uses rake). Gemika does not need to know about rspec. Only the Gemfiles managed by gemika with use spreewald need to know about rspec (and your commit introduces that :) ).